### ISSUE TITLE
Failing to marshal Href Attributes

### ISSUE THREAD

**Issue by minhaj10p** [opencontrol/oscalkit#8]
oscalkit failing to convert NIST80053 catalog rev4 json file
Link:
https://github.com/usnistgov/OSCAL/blob/master/content/nist.gov/SP800-53/rev4/NIST_SP-800-53_rev4_declarations.json
Error:
Its unable to marshal all the Href attributes
```
type Href struct {
	*url.URL
}
```

**Comment by anweiss** [comment]
Good catch @minhaj10p. Of note, the JSON samples in https://github.com/usnistgov/OSCAL are slightly out of date from the latest schemas. So you may come across a few other JSON marshaling failures.
